examples/batch_processing/test/data.csv

```diff
@@ -0,0 +1,150 @@
+5.1, 3.5, 1.4, 0.2
```

Comment:
Ugh this seems like the dataset from iris dataset, so why should we check this in?

Reply:
This is just local testing code, it'll be gone when the PR is ready.